.travis.yml

```diff
@@ -13,6 +13,14 @@ jdk:
 script:
   - ./gradlew clean build check
 
+after_success:
+  - .buildscript/deploy_snapshot.sh
+
+env:
+  global:
+    - secure: "fqr1/N5DjhquxDP+F15OfhD58UBaORHvv/ReNKBIOrsvrJLn5FIzBW+enljZxLpgo3mUAJo8xfAo0IVM35fEXzRg/RZUkir27M62UGvGCATV4HX4NnUpzvYnyxzuwRIM5gTb9TzcQughEG891tgwxfIGloMeUGeoMMpogQvx3oU1186DFNRw/gAv9Qk7lFnVFd2/euvc4CwZQQdsvNvTSOxvD4Uy+/po9hYaQwPPkBTOCabUuZgyvGcHSQOVHEu9WbQocwp3qyQ1UVS+auTBucumGvIH2ivK/j3MKqcPWEws9H5iJCwBkWdWm1rcIHXORW+S25DcgJpztoeWYre0Tr0w+CVGarnwLOBWJPEL2oM6LS9MSrg6Ub6cTBrDHBJS5/j3bkYZ9ZtB1/zct4z7Opox0PanfxBddNcoOgAw7QvfZH5MU+x3TTN374+NHfKNcQz83nqJ5oHEOFTjo4MtOjtZsHGCrKf8iXrS2gM9CCYE3jkZTNaAJwNRsqbTx7xEH568UHXPuWLpoYaCRtkCehMkebH6ZITcJ/1fQkXYrg376jbLbsgq8GcYjRONc+QAasclmjZ8HC9wCAVU3Ul8hQUIUkaM9a561ROP8z9KGkS8pc6oFdzV2Tdodkx8BDSnjnZ/ELLCcub8q2+aw7GzYZgnM1RZqfyOjtJnYaBpxkc="
+    - secure: "sH5NFiUeMv0ndzwNMU+eUwdL4BSRYTTiFupR3BdW6g2NGw7X0XtRolGsNgaAQPnvVWruaxYwp6X4M+4nhTTN4786uSXzNdP9Jmt12HpDT7SZ5+Xi5O0Z8zwfj+QsQCDVfxjZXIv8aN8xfcbgFwOkV9yMZGQsvupZ1zLSJcO0lUXVTblX5xM7s+KDaptirBk/NEZEvPEOkU9QPztty9J435jLXHNDbU8WSYcCWlKGFLztpGmL6BLFXi/gfR8nJ5gGRTibhYr0+vbiuRXJKiwKf66fplqsdQod7NX//FLouL1UEzi9pO348wXv9nKYjgrRiqiR27OUMt1z2LJodQ4npBSIWmJ75TYIq7oGDlxGipkviBQxXyy6zE5Ifqw8amHtFNJbReApZBPMLAOGGINtOAlMvka47uzsAKNemdf8D54B9W2aCA7G3taH3SAAye+HGEXwF8/HElxQi7uxy60j1RsTu2AG7mvrIrTqY8ZE/TeCeksNr0d45wa9QjksM+1z2pweQSKMcJmD0zmSr0tTyaRFkcDf2dF/rZhYd5yvaJS69hXM8taLshP0lDrCGlC/mjuzmfhtp07dcYB5+F3HRSEIxDF9HIQto5cJIb6THFInWOYQ3XohaQLs8gDqtZU0Z1f9c9Uhh8/qurvsnkzgp8Gred7R8PYs8LtAlT9q54s="
```

Comment:
Why not? I generated new ones with my own credentials for square/leakcanary. Happy to use the bot ones if you provide them :)